src/register/tests/RegistrationPage.test.jsx

```diff
@@ -1041,7 +1043,7 @@ describe('RegistrationPage', () => {
       getLocale.mockImplementation(() => ('ar-ae'));
 
       const registrationPage = mount(reduxWrapper(<IntlRegistrationPage {...props} />));
-      registrationPage.find('input#country').simulate('focus');
+      registrationPage.find('input[name="country"]').simulate('click');
```

Comment:
Can you please explain why do we need to change the event type ? is it because of paragon dropdown uses?

Reply:
Sorry for misunderstanding the question before^.

Yes, you are right. It's because of the paragon component. They open the dropdown on `onClick` event.